### PR TITLE
JPN-432 Use throbber as loading indicator for modal

### PR DESCRIPTION
**Pull request by perjg**
As all modal data is cached and prefetched, it is difficult to test this directly.

The easiest way to test is to edit ext.communityPage.js and comment out lines 195-197.

getModalTabContentsHtml(tabs.TAB_ALL);
getModalTabContentsHtml(tabs.TAB_ADMINS);
getModalTabContentsHtml(tabs.TAB_LEADERBOARD);

Also, to disable caching add the following new line at line 89:

data: {mcache: 'writeonly'},

### CODE REVIEW

File: extensions/wikia/CommunityPage/scripts/ext.communityPage.js [hunk 1]
```diff
@@ -124,16 +125,16 @@ require([
 				}
 			};
 			uiModal.createComponent(createPageModalConfig, function (modal) {
-				var loading = mustache.render(templates.modalLoadingScreen, {
-						loadingText: $.msg('communitypage-modal-tab-loading'),
-					}),
+				var loading = mustache.render(templates.modalLoadingScreen),
 					html = navHtml + loading;
```
Comment (kamilkoterba) [inline extensions/wikia/CommunityPage/scripts/ext.communityPage.js]: We can remove loading and html variables and construct when calling html() as it's used only there.

File: extensions/wikia/CommunityPage/templates/modalLoadingScreen.mustache [hunk 2]
```diff
@@ -1,3 +1 @@
-<p>
-	{{loadingText}}
-</p>
+<div class="throbber"></div>
```
Comment (kamilkoterba) [inline extensions/wikia/CommunityPage/templates/modalLoadingScreen.mustache]: Minor. I'd call it throbber-placeholder. Does that sound good to you?

**Comment by kamilkoterba** [comment]
Two minor comments.
LGTM.

**Comment by perjg** [comment]
@kamilkoterba Cleaned up according to both your comments.

**Pull request merged**